ui/raidboss/timeline.ts

```diff
@@ -424,8 +523,8 @@ export class Timeline {
     });
   }
 
-  Stop() {
-    this.timebase = null;
+  public Stop(): void {
+    this.timebase = 0;
```

Comment:
I think this seems reasonable.  Stopped is still `0`, and there's nothing special about `null` here.

Reply:
I had some concerns about it still working with the raidboss emulator, but I just tested it and it seems to be working as expected.